vic/drivers/shared_image/src/vic_init.c

```diff
@@ -986,8 +986,7 @@ vic_init(void)
                 }
                 sum += soil_con[i].AreaFract[j];
             }
-            // TBD: Need better check for equal to 1.
-            if (sum != 1.) {
+            if (!assert_close_double(sum, 1.0, 0., 0.001)) {
```

Comment:
I guess option 3 allows for the simplest, easiest-to-maintain code.  But it does place the burden on the user - is it possible to have a case where a user could have real trouble getting fractions to sum to within 1e-20 of 1.0 in their netcdf file?  Or perhaps a better question is: would it be common enough for this to happen that VIC should be more accommodating?